docs/configuration/README.md

```diff
@@ -10,7 +10,7 @@ You can check what version of EventStoreDB you have installed by using the `--ve
 
 ```bash
 $ eventstore --version
-EventStoreDB version v21.2.0 (tags/oss-v21.2.0/bc30009, Unknown)
+EventStoreDB version v21.6.0 (tags/oss-v21.6.0/bc30009, Unknown)
```

Comment:
**SUGGESTION:** It would be good to update with the proper hash once the 21.6 is released.